src/StopwatchPlugin.php

```diff
@@ -39,11 +40,11 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
         $this->stopwatch->start($eventName, self::CATEGORY);
 
         return $next($request)->then(function (ResponseInterface $response) use ($eventName) {
-            $this->stopwatch->stop($eventName, self::CATEGORY);
+            $event = $this->stopwatch->stop($eventName);
 
-            return $response;
+            return $response->withHeader(self::HEADER, $event->getDuration());
```

Comment:
Hm, so the problem is that we cannot map a request to a specific stopwatch event, right?

Because my idea was to use the stopwatch class and get the event based on something: http://api.symfony.com/3.2/Symfony/Component/Stopwatch/Stopwatch.html#method_getEvent

Maybe we could introduce an internal Request/Response ID header and use that in the event name, but that still feels hacky.